src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -245,7 +245,7 @@ public sealed class WordHashBagEstimator : TrainedWrapperEstimatorBase
         public override TransformWrapper Fit(IDataView input)
         {
             // Create arguments.
-            var args = new WordHashBagProducingTransformer.Arguments
+            var args = new WordHashBagProducingTransformer.Options
```

Comment:
>args [](start = 16, length = 4)

Could you rename to options?

Reply:
fixed

---
In reply to: [254075307](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254075307) [](ancestors = 254075307)